### ISSUE TITLE
Support for Equilibrium Level 0

### ISSUE THREAD

**Issue by lexisother** [Melledy/LunarCore#60]
It seems that enemies are all above level 30 despite the Equilibrium level being 0, 30 is the minimum level for Equilibrium level 1, so is there any chance support for level 0 will be added?

**Comment by GorujoCY** [comment]
oh look
https://bw.artemislena.eu/honkai-star-rail/wiki/Equilibrium_Level#Level_1

it's called
DO SOME RESEARCH

but yes no, the minimum equilibrium level 1 and the max is 6, not 0

**Comment by lexisother** [comment]
Fair enough! I was just wondering if it's something anyone would be willing to look into because for someone like me who likes to see stuff scale from nothing, it's nice to test things out. I could always look into how Equilibrium stuff is implemented in the server and see if I can work it out myself.

**Comment by Melledy** [comment]
Monster levels in battle are calculated client side from stage data in the client. If you're in a zone where monsters are over level 30 at EQ 0, then chances are that the zone wasnt meant to be played in at EQ 0 on the official server.

**Comment by lexisother** [comment]
It's Belobog, and as far as I remember monsters shouldn't be level 36 there. You only start increasing Equilibrium after you defeat Svarog IIRC.
